boards/arm/arduino_nano_33_ble/doc/index.rst

```diff
@@ -121,6 +121,30 @@ Then, you can flash the image using the above script.
 
 You should see the the red LED blink.
 
+Debugging
+=========
+
+You can debug an application on the board with a debug adapter that supports
+CMSIS-DAP. This board has the SWD connector for debugging but exposes it as
+a test pad pattern (not a connector) on the back side of the PCB. So, It needs
+to be a bit difficult soldering. At least, the SWDIO and the SWCLK need
+to soldering. (As shown in the picture.)
+GND, 3.3V, and RESET are also available in the DIP connector. Therefore, it may be
+easier to connect these.
```

Comment:
```suggestion
bit of difficult soldering. At a minimum, SWDIO and SWCLK need soldering (As
shown in the picture). GND, 3.3V, and RESET are also available in the DIP
connector, therefore it may be easier to connect using the DIP connector
instead of soldering to them.
```

Reply:
Applied it.
Thank you for correcting.